src/volta-shim.rs

```diff
@@ -1,5 +1,38 @@
-mod execute;
+#![cfg(feature = "volta-updates")]
```

Comment:
So, once we un-flag `volta-updates`, we will delete `shim.rs`?

Reply:
Yeah, that's the plan.